compose/cli/command.py

```diff
@@ -110,13 +107,34 @@ def get_config_path(self, file_path=None):
         if file_path:
             return os.path.join(self.base_dir, file_path)
 
-        if os.path.exists(os.path.join(self.base_dir, 'docker-compose.yaml')):
-            log.warning("Compose just read the file 'docker-compose.yaml' on startup, rather "
-                        "than 'docker-compose.yml'")
+        supported_filenames = [
+            'docker-compose.yml',
+            'docker-compose.yaml',
+            'fig.yml',
+            'fig.yaml',
+        ]
+
+        def expand(filename):
+            return os.path.join(self.base_dir, filename)
+
+        candidates = [filename for filename in supported_filenames if os.path.exists(expand(filename))]
+
+        if len(candidates) == 0:
+            raise errors.ComposeFileNotFound(supported_filenames)
+
+        winner = candidates[0]
+
+        if len(candidates) > 1:
+            log.warning("Found multiple config files with supported names: %s", ", ".join(candidates))
+            log.warning("Using %s\n", winner)
+
+        if winner == 'docker-compose.yaml':
             log.warning("Please be aware that .yml is the expected extension "
                         "in most cases, and using .yaml can cause compatibility "
-                        "issues in future")
+                        "issues in future.\n")
 
-            return os.path.join(self.base_dir, 'docker-compose.yaml')
+        if winner.startswith("fig."):
+            log.warning("%s is deprecated and will not be supported in future. "
+                        " Please use docker-compose.yml from now on.\n" % winner)
```

Comment:
nit: perhaps "Please rename your config file to `docker-compose.yml`" (or similar)?